Header header

float64 boomV
float64 armV
float64 bucketV
float64 swingV
float64 boomP
float64 armI
bool Kmode
